### ISSUE TITLE
Lucky Block V-11 Forge doesn't work with Forge 1.18.2

### ISSUE THREAD

**Issue by KristenStuffs** [alexsocha/luckyblock#67]
I was working on a small update for my lucky block add-on and I noticed the curseforge page for the lucky block mod under files said Version 11 worked with Minecraft Forge 1.18.2. After installing forge I pressed play, and then the games loads to a certain point before saying the lucky block mod has failed to load correctly which results in a crash.

Operating System: Windows 10.
Minecraft Version: 1.18.2
Lucky Block Version: Forge-11.
Forge version: 1.18.2 - 40.0.2.

Crash Report:
[crash-2022-03-03_02.50.06-fml.txt](https://github.com/alexsocha/luckyblock/files/8175830/crash-2022-03-03_02.50.06-fml.txt)

**Comment by Brandonbr1** [comment]
> Forge-11

seems to be an issue with forge itself not the mod try reinstalling forge to the latest version or if that doesn't work reinstall Minecraft

let me know how that goes

**Comment by KristenStuffs** [comment]
Just tried reinstalling forge, didn't work. Tried a brand new instance of minecraft which is separate from my main instance (via the curseforge launcher). Both of with resulted in the same error I had gotten before. I even tried without any of my add-ons I had been working on which resulted in the same error.

I tried several other mods by themselves (or with their dependencies) including: Just Enough Items, Repurposed Structures, and Enchant With This Mob all of which did not result in a crash or error upon loading.
